src/View/printErrorMessageForInputEmpty.js

```diff
@@ -0,0 +1,5 @@
+const printErrorMessageForInputEmpty = () => {
+  throw new Error('[ERROR] 아무것도 입력하지 않으셨습니다. 프로그램을 종료하겠습니다.');
+};
+
+export default printErrorMessageForInputEmpty;
```

Comment:
아래의 에러메세지 출력 코드도 지금 코드랑 로직이 똑같은 것 같아요 하나의 템플릿으로 만들어서 관리하는 건 어떨까요?

```suggestion
const printErrorMessage = message => {
  throw new Error(`[ERROR] ${message}`);
};
```